lib/index.js

```diff
@@ -355,6 +385,10 @@ api.verify = async ({
     verified = crypto.timingSafeEqual(
       new Buffer(token.sha256, 'base64'),
       new Buffer(prefixedHash(hash), 'base64'));
+  } else if(token.authenticationMethod === 'totp') {
```

Comment:
```suggestion
  } else if(token.type === 'totp') {
```

Reply:
`token` is the token that comes out of the database which has `authenticationMethod` as distinct from the `totpToken` that is passed into the API.

Reply:
Fixed.